bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIComponentSitemapProvider.java

```diff
@@ -403,10 +298,10 @@ private void addWidgetMappings(EList<Mapping> mappings, UIComponent component) {
                         releaseCmd = stripQuotes(releaseCmd);
                         String label = stripQuotes(splitMapping[1].trim());
                         String icon = splitMapping.length < 3 ? null : stripQuotes(splitMapping[2].trim());
-                        MappingImpl mapping = (MappingImpl) SitemapFactory.eINSTANCE.createMapping();
-                        mapping.setCmd(cmd);
+                        Mapping mapping = sitemapFactory.createMapping();
+                        mapping.setCmd(cmd != null ? cmd : "");
                         mapping.setReleaseCmd(releaseCmd);
-                        mapping.setLabel(label);
+                        mapping.setLabel(label != null ? label : "");
```

Comment:
For consistency, you could do the same as in addWidgetButtons, that is testing if value is not null before calling setCmd and setLabel.